index.js

```diff
@@ -29,18 +60,23 @@ function UI() {
 
 module.exports = {
   UI: UI(),
-  setQueues: (bullQueues) => {
+  setQueues: bullQueues => {
     if (!Array.isArray(bullQueues)) {
-      bullQueues = [bullQueues];
+      bullQueues = [bullQueues]
     }
 
-    bullQueues.forEach((item) => {
-      queues[item.name] = item;
+    bullQueues.forEach(item => {
+      queues[item.name] = item
     })
 
     return queues
   },
   createQueues: redis => {
+    if (isBullMQ) {
```

Comment:
I've been actually considering removing the option to `createQueues` and keep only `setQueues`, what do you think? 

this way we can keep the lib easier to maintain! 👍

Reply:
👍 Make sense, I'll remove it

Reply:
Just realize that it was the only place where we needed to `require('bull')`. It means that we can remove bull and bullmq from peerDependencies, right?

Reply:
yes! exactly, this way we don't need to control the lib version or anything, leaving it only up to the users and making sure we support the correct versions 😄

Reply:
After some testing, I saw that some functions have changed signature between Bull3 and BullMQ...

I can detect the version of a queue by checking (for example) if the `drain` method exists (BullMQ) or not (Bull3). This way, we can switch if needed...

For example, `queue.clean` has not the same signature:
```js
// With BullMQ
await queue.clean(GRACE_TIME_MS, LIMIT, queueStatus)
// With Bull3
await queue.clean(GRACE_TIME_MS, queueStatus)
```

What do you think?

Reply:
yeah, that's tricky 😅 
I would say we can have some check when constructing the lib, and have that variable in the locals, I guess it will be a consequence to have some conditions in some places

Reply:
I added a map to store the version for each queue (this way a system can have both versions simultaneously). The function that check the bull version is [here](https://github.com/vcapretz/bull-board/pull/49/files#diff-168726dbe96b3ce427e7fedce31bb0bcR39)